Amasty_Gdpr/templates/checkboxNewsletter.phtml

```diff
@@ -40,12 +40,12 @@
         >
             <use xlink:href="<?= $block->escapeUrl($block->getViewFileUrl('/images/icons-sprite.svg#unchecked')) ?>"></use>
         </svg>
-        
+
         <label
             for="amgdpr_agree_newsletter"
             class="checkbox__label"
         >
-            <?= $block->getConsentText() ?>
+            <?= $block->escapeHtml($block->getConsentText()) ?>
```

Comment:
In original file I see: <?= $block->escapeHtml($block->getConsentText($consent), ['a']) ?>